src/gui/widgets/TabWidget.cpp

```diff
@@ -392,6 +398,18 @@ void TabWidget::setTabSelected( const QColor & c )
 	m_tabSelected = c;
 }
 
+// Return the text color of the selected tab
+QColor TabWidget::tabTextSelected() const
+{
+	return m_tabTextSelected;
+}
+
+// Set the text color of the selected tab
+void TabWidget::setTabTextSelected( const QColor & c )
```

Comment:
```suggestion
void TabWidget::setTabTextSelected(const QColor & c)
```

Reply:
Spaces fixed with commit 0fa18d405e7 where I have fixed style issues for all of `TabWidget`.